src/pages/page-2.js

```diff
@@ -1,16 +1,12 @@
 import React from 'react';
 import { Link } from 'gatsby';
-
+import Contentp2 from '@sections/contentp2';
 import Layout from '@common/Layout';
 import { Container } from '@components/global';
 
 const SecondPage = () => (
   <Layout>
-    <Container>
-      <h1>Hi from the second page</h1>
-      <p>Welcome to page 2</p>
-      <Link to="/">Go back to the homepage</Link>
-    </Container>
+    </Contentp2>
```

Comment:
This should be `<Contentp2 />` instead of what you have now, React self closing components have their forward slash at the end rather than at the begining like html elements. This is why your build was failing.

I haven't checked your code fix yet and that would be reviewed by @apollonian .

Reply:
I just followed the same syntax of Header in previous page.
That's the reason I followed this.